TUTORIAL.md

```diff
@@ -117,25 +117,25 @@ from dataflows import Flow, printer
 from xml.etree import ElementTree
 from urllib.request import urlopen
 
-# Get from Wikipedia the population count for each country
+
 def country_population():
+    """Get from Wikipedia the population count for each country."""
```

Comment:
Changes made to the Python code in the notebook were replicated in the tutorial too.